frontend/src/model/AccessRequests.tsx

```diff
@@ -23,12 +24,16 @@ export default function AccessRequests({ model }: AccessRequestsProps) {
         accessRequests
           .sort(sortByCreatedAtDescending)
           .map((accessRequest) => (
-            <AccessRequestDisplay accessRequest={accessRequest} key={accessRequest.metadata.overview.name} />
+            <AccessRequestDisplay
+              accessRequest={accessRequest}
+              key={accessRequest.metadata.overview.name}
+              hideReviewBanner={!roles?.some((role) => ['msro', 'mtr'].includes(role))}
```

Comment:
Can we move this logic into `frontend/utils/roles.ts` as a reusable function?
```
export const hasRole = (userRoles: string[], validRoles: string[]) => {
 return userRoles.some((role) => validRoles.includes(role))
}
```
You can then use it here like so:
```
hideReviewBanner={!hasRole(roles, ['msro', 'mtr'])}
```